src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -149,8 +149,16 @@ public static Attendance parseAttendance(String input) throws ParseException {
         try {
             return Attendance.fromDateString(input);
         } catch (IllegalArgumentException e) {
-            throw new ParseException(Attendance.MESSAGE_CONSTRAINTS);
+            throw parseException(input, Attendance.MESSAGE_CONSTRAINTS);
         }
     }
 
+    private static String inputIsWrong(String input, String message) {
```

Comment:
Want to consider renaming the method name, to sound more like an actionable verb i.e `returnsWrongInput`? Just to match the current code quality standards at https://se-education.org/guides/conventions/java/basic.html#naming?